### DISCUSSION TITLE
Parent/child selection box fiter options

### DISCUSSION THREAD

**Discussion by tuluong-vl** [rappasoft/laravel-livewire-tables#385] [Q&A] [answered]
Hi @rappasoft 

Does the current version have option to generate dynamic options for Select filter Type? I need  to have parent / child selection box for filter options.

If not, please suggest me a solution to do this.

Thanks a lot.

**Comment by rappasoft** [comment, marked as answer]
I don't think I follow fully but regardless I don't think there's a built-in option.
